lib/compiler/src/translator/module.rs

```diff
@@ -84,33 +84,36 @@ pub fn translate_module<'data>(
             }
 
             Payload::InstanceSection(_)
-            | Payload::AliasSection(_)
-            | Payload::ModuleSectionStart { .. }
-            | Payload::ModuleSectionEntry { .. } => {
+            | Payload::ComponentSection { .. }
+            | Payload::CoreTypeSection(_)
+            | Payload::ComponentTypeSection(_)
+            | Payload::ComponentInstanceSection(_)
+            | Payload::ComponentAliasSection(_)
+            | Payload::ComponentCanonicalSection(_)
+            | Payload::ComponentStartSection { .. }
+            | Payload::ComponentImportSection(_)
+            | Payload::ComponentExportSection(_)
+            | Payload::ModuleSection { .. } => {
                 unimplemented!("module linking not implemented yet")
```

Comment:
Component Types / Module linking shit.

```suggestion
                unimplemented!("module linking not implemented. It will only be implemented if/when browsers support it")
```